combat/advancedCombat/readme.js

```diff
@@ -0,0 +1,43 @@
+// How to use advancedCombat
```

Comment:
Denne filen burde vel egentlig være en .txt eller .md fil for å gjøre det mer forståelig at det er en readme som ikke har noen funksjoner e.l. i seg